scripts/ci/manage-operator.sh

```diff
@@ -44,7 +44,7 @@ push_image() {
     GIT_COMMIT_ID=$(git --git-dir=${REPOSITORY_PATH}/.git --work-tree=${REPOSITORY_PATH} rev-parse --short HEAD)
     IMAGE_LOC=quay.io/codeready-toolchain/${REPOSITORY_NAME}:${GIT_COMMIT_ID}
     if is_provided_or_paired; then
-        make -C ${REPOSITORY_PATH} docker-push QUAY_NAMESPACE=${QUAY_NAMESPACE} IMAGE_TAG=${TAGS}
+        make -C ${REPOSITORY_PATH} ${IMAGE_BUILDER}-push QUAY_NAMESPACE=${QUAY_NAMESPACE} IMAGE_TAG=${TAGS}
```

Comment:
the IMAGE_BUILDER variable is not set for this script - it's only used & set in CD scripts, not in CI. 
You could for example do:
```suggestion
        IMAGE_BUILDER=${IMAGE_BUILDER:-"podman"}
        make -C ${REPOSITORY_PATH} ${IMAGE_BUILDER}-push QUAY_NAMESPACE=${QUAY_NAMESPACE} IMAGE_TAG=${TAGS}
```
or add a new flag to the manage-host/member-operator scripts

Reply:
thanks for your suggestion, I've applied it as it seems like the most straightforward way to fix the problem (at least to me, for now)